ah.mjs

```diff
@@ -1,43 +0,0 @@
-import playwright from 'playwright';
-import { PlaywrightCrawler, ProxyConfiguration } from './packages/playwright-crawler/dist/index.mjs';
-
-const proxyConfiguration = new ProxyConfiguration({
-    proxyUrls: [
-        'http://auto:ot4HQbTnKC3rMHpL4LJJWw6ni@proxy.apify.com:8000',
```

Comment:
I accidentally committed my proxy password, but I reset it so it's not working anymore 😅